amis/windows/windows-docker.json

```diff
@@ -36,7 +36,8 @@
         "amis/windows/scripts/docker.ps1",
         "amis/windows/scripts/windows-install-packages.ps1",
         "amis/windows/scripts/windows-nuget.ps1",
-        "amis/windows/scripts/buildtools2017.ps1"
+        "amis/windows/scripts/buildtools2017.ps1",
+        "amis/windows/scripts/net-reference-assemblies.ps1"
       ],
```

Comment:
Did you need to add 2019 here?

Reply:
No, this is the old 2017 script, I was just keeping it updated for the file splits.